db/seeds.d/101-locations.rb

```diff
@@ -4,7 +4,7 @@
 # !!! PLEASE KEEP THIS SCRIPT IDEMPOTENT !!!
 #
 
-if Location.exists? &&
+if Location.exists? && (ENV['SEED_LOCATION'].present? || Location.first.present?) &&
```

Comment:
Should the last two conditions be lumped together into a parenthesis statement? Based on the current logic, all of these ands at the beginning look like they can be short circuited by the last condition leading errors that this new condition is trying to prevent.

Reply:
Good point @ehelms updated!